app/models/user.rb

```diff
@@ -1,8 +1,14 @@
 class User < ActiveRecord::Base
   has_many :videos
-  has_many :queue_items
+  has_many :queue_items, -> {order "position ASC"}

```

Comment:
You do not have to supply "ASC" to position, because that is just the default order. :)